xml/System.IO/FileSystemInfo.xml

```diff
@@ -657,13 +657,18 @@ On Unix platforms that do not support creation or birth time, this property retu
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the string representing the extension part of the file.</summary>
+        <summary>Gets the extension part of the file name, including the leading dot <c>'.'</c> - or an empty string if no extension is present - or the file's <see cref="FileSystemInfo.Name"/> if dot-prefixed with no other dots present.</summary>
```

Comment:
```suggestion
        <summary>Gets the extension part of the file name, including the leading dot <c>.</c>, or an empty string if no extension is present.</summary>
```

Since the dot-prefixed file name is covered in the remarks and it makes the summary perhaps too verbose, I removed it here.

Reply:
How about "Gets the extension part of the file name, including the leading dot <c>.</c> even if it is the entire file name, or an empty string if no extension is present." ?

Reply:
I've made the summary much shorter but included the bit about matching the whole file-name without being verbose.